### DISCUSSION TITLE
Is it maintained? Or should i use this one until i migrate dgrijalva/jwt-go?

### DISCUSSION THREAD

**Discussion by mahesh-riddles** [golang-jwt/jwt#314] [Q&A] [answered]
I am confused about this package.
I was dgrijalva/jwt-go. 

Sometime i stopped working on go.
Now i can see dgrijalva/jwt-go has been moved to new domain.

Is it maintained or is it to give time to migrate from dgrijalva/jwt-go?
Please someone respond.

Thanks
Mahesh Kumar

**Comment by oxisto** [comment, marked as answer]
> I am confused about this package. I was dgrijalva/jwt-go.
> 
> Sometime i stopped working on go. Now i can see dgrijalva/jwt-go has been moved to new domain.
> 
> Is it maintained or is it to give time to migrate from dgrijalva/jwt-go? Please someone respond.
> 
> Thanks Mahesh Kumar

Hi,

Yes this project is maintained by @mfridman and myself. We also just recently released [v5.0.0](https://github.com/golang-jwt/jwt/releases/tag/v5.0.0). You can find some more information on how to migrate from dgrijalva/jwt-go here: https://github.com/golang-jwt/jwt/blob/main/MIGRATION_GUIDE.md. If you are just using the library, most of the migration should be relatively straightforward.

**Reply by maheshkumar2150** [reply]
Thanks a lot for the information.. You are awesome.. Thanks for helping people..
